src/Documentation/sidebar.json

```diff
@@ -23,34 +23,26 @@
       "example-pipeline.md"
     ],
     "labels": {
-      "connect-code-and-data.md": "Connect with Code",
-      "example-pipeline.md": "Example: Pipelines",
-      "example-versioning.md": "Example: Versioning",
-      "older-versions.md": "Get Older Files"
+      "agenda.md": "Agenda"
     }
   },
   {
     "name": "Use Cases",
     "folder": "/static/docs/use-cases",
-    "indexFile": "index.md",
+    "indexFile": "use-cases.md",
```

Comment:
why rename it?

Reply:
because url is formed in the file name

Reply:
how did it work before? what has changed?

Reply:
there was no need to put the same name for the index file as for the directory it is contained in. So why don't we just keep `index.md` everywhere as it was before?

Reply:
because there used to be a different json file structure